azure/table_azure_app_service_web_app.go

```diff
@@ -293,6 +306,29 @@ func getAppServiceWebAppSiteAuthSetting(ctx context.Context, d *plugin.QueryData
 	return op, nil
 }
 
+func getAppServiceWebAppVnetConnection(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Comment:
> select * from azure_app_service_web_app

Error: web.AppsClient#GetVnetConnection: Failure responding to request: StatusCode=200 -- Original Error: Error occurred unmarshalling JSON - Error = 'json: cannot unmarshal array into Go value of type map[string]*json.RawMessage' JSON = '[]'

+----------+----------------------------------------------------------------------------------------------------------------+------+---------+---------------------+--------------
| name     | id                                                                                                             | kind | state   | type                | client_affini
+----------+----------------------------------------------------------------------------------------------------------------+------+---------+---------------------+--------------
| test5666 | /subscriptions/d46d7416-f95f-4771-bbb5-529d4c76659c/resourceGroups/demo/providers/Microsoft.Web/sites/test5666 | app  | Running | Microsoft.Web/sites | true         
|          |                                                                                                                |      |         |                     |              
+----------+----------------------------------------------------------------------------------------------------------------+------+---------+--------------------+--------------
Getting this error when we have web app without any vnet connection. Please check.